### PR TITLE
Added condefusion benchmarks

### PR DESCRIPTION
**Pull request by MukulSingh105**

### CODE REVIEW

File: CodeFusion/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# CodeFusion Data and Benchmarks
+
+This repository contains the conditional formatting benchmarks used in CodeFusion (CodeFusion: A Pre-trained Diffusion Model for Code Generation)
+
+## Folder Structure
+
+- `Benchmarks`
+    - Training and Testing data for Python, M and CF Rules
+- `Pretraining Data`
+    - Unannotated code used for pretraining for Python, M and CF Rules
```
Comment (josepablocam) [inline CodeFusion/README.md:10]: right now I only see Bash. We should amend this. If we are not releasing CF due to bing21 issues, then let's just state we cannot due to compliance.

File: CodeFusion/README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# CodeFusion Data and Benchmarks
+
+This repository contains the conditional formatting benchmarks used in CodeFusion (CodeFusion: A Pre-trained Diffusion Model for Code Generation)
+
+## Folder Structure
+
+- `Benchmarks`
+    - Training and Testing data for Python, M and CF Rules
```
Comment (josepablocam) [inline CodeFusion/README.md:8]: I think you meant Bash

**Review by josepablocam — CHANGES_REQUESTED** [review]

File: CodeFusion/README.md (hunk 1 above)
Comment (MukulSingh105) [reply CodeFusion/README.md:10]: Added now.

File: CodeFusion/README.md (hunk 2 above)
Comment (MukulSingh105) [reply CodeFusion/README.md:8]: Added others too now.

**Review by josepablocam — APPROVED** [review]

**Pull request merged**